### DISCUSSION TITLE
Push map changes to Deebot?

### DISCUSSION THREAD

**Discussion by headart** [mrbungle64/node-red-contrib-ecovacs-deebot#71] [Q&A] [answered]
I'm not sure if anyone else is having this issue but my T8 AIVI always seems to update the map. This means that I cannot reliably use area names as there's a high chance that these have changed.

Has anyone come across a way to overcome this issue or are we able to push an edited map back to the Deebot for it to use?

I can imagine pushing a map back to the deebot to function where we download it when we are happy with the map and then upload it when needed. I know there is a restore function within the map but this only takes it back to the initial state after the first Automap.

**Comment by mrbungle64** [comment, marked as answer]
@headart 

This problem is not related to the Ecovacs Deebot node for Node-RED, right?

> or are we able to push an edited map back to the Deebot for it to use?

I am not aware of any way to push a backed up map to the Deebot.

Btw.: I also own a Deebot OZMO T8 AIVI, but I haven't had this problem yet.

**Reply by headart** [reply]
Not specifically but was more just setting the scene for the question. 
Thanks anyway.
